collectfast/management/commands/collectstatic.py

```diff
@@ -32,14 +32,7 @@ def __init__(self, *args, **kwargs):
         self.tasks = []
         self.etags = {}
         self.collectfast_enabled = settings.enabled
-
```

Comment:
This code will trigger the storage class' `__init__()` function, which in turn triggers `check_preload_metadata()`, which contains this same code without the `is_boto(...)` statement